rhel/vex/parser.go

```diff
@@ -100,7 +100,12 @@ func (u *Updater) DeltaParse(ctx context.Context, contents io.ReadCloser) ([]*cl
 		}
 	}
 	vulns := []*claircore.Vulnerability{}
-	for _, vs := range out {
+	for n, vs := range out {
+		if len(vs) == 0 {
+			// If there are no vulns for this CVE make sure we signal that
+			// it is deleted in case it once had vulns.
+			deleted = append(deleted, n)
```

Comment:
nit: `continue`?

Reply:
errrm, I guess it would guard against anything else being hypothetically added after appending to the vulns but then again it's hard to know if we'd want that hypothetical thing to run anyway. I guess I'll add it